### DISCUSSION TITLE
Is there no more support for Angular?

### DISCUSSION THREAD

**Discussion by Darioba1987** [nolimits4web/swiper#6369] [Q&A]
Maybe I'm crazy, but a few weeks ago I installed Swiper in my new angular project, today I wanted to check something and I saw that there is no documentation for angular. Does this mean that there is no more support?

Thanks :)

**Comment by Arhell** [comment]
https://github.com/nolimits4web/swiper/blob/master/CHANGELOG.md#900-2023-02-01
v9

_removed Swiper Angular components (in favor of Swiper WebComponent)_

**Comment by LukasKlement** [comment]
Angular supports Web Components, a clean way to use Swiper in Angular is to create a directive as detailed in this SO post: https://stackoverflow.com/a/75353889/3685290
